.drone.yml

```diff
@@ -780,9 +780,9 @@ steps:
       # increment these variables when a new major/minor version is released to bump the automatic builds
       # this only needs to be done on the master branch, as that's the branch that the Drone cron is configured for
       # build major version images which are just teleport:x
-      CURRENT_VERSION_ROOT: v14
-      PREVIOUS_VERSION_ONE_ROOT: v13
-      PREVIOUS_VERSION_TWO_ROOT: v12
+      CURRENT_VERSION_ROOT: v15
```

Comment:
FYI I have a PR in the merge queue that will remove the need for this + the dronegen changes.

Reply:
I'll rebase once your PR lands.

Reply:
Sorry yes this is still required (for now). I pulled out all the container image pipelines but this one.